eiffel-vocabulary/EiffelIssueDefinedEvent.md

```diff
@@ -16,18 +16,15 @@
 --->
 
 # EiffelIssueDefinedEvent (ID)
-The EiffelIssueDefinedEvent declares that an issue has been created in some
-external issue management software. ID is semantically similar to
-[EiffelFlowContextDefinedEvent](../eiffel-vocabulary/EiffelFlowContextDefinedEvent.md)
-and [EiffelEnvironmentDefinedEvent](../eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md).
+The EiffelIssueDefinedEvent declares that an issue has been created in someexternal issue management software. ID is semantically similar to [EiffelFlowContextDefinedEvent](../eiffel-vocabulary/EiffelFlowContextDefinedEvent.md) and [EiffelEnvironmentDefinedEvent](../eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md).
```

Comment:
```suggestion
The EiffelIssueDefinedEvent declares that an issue has been created in some external issue management software. ID is semantically similar to [EiffelFlowContextDefinedEvent](../eiffel-vocabulary/EiffelFlowContextDefinedEvent.md) and [EiffelEnvironmentDefinedEvent](../eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md).
```

Reply:
There are multiple whitespace issues in this file, emerging from the yaml files. Please update those.

Reply:
Fixed in 888fdaa893e88ed28b6374edf2a6aa24539ccfec.

Reply:
Some more whitespace errors spotted. Commented separately.